app/hotels/src/SingleHotel.js

```diff
@@ -5,14 +5,56 @@ import { View, Text } from 'react-native';
 import { Button } from '@kiwicom/native-common';
 
 type Props = {
-  onGoToHotelGallery: () => void,
+  onGoToHotelGallery: (
+    hotelName: string,
+    images: Array<{|
```

Comment:
this might use definition of `Image` from GalleryGrid

Reply:
I was thinking about this but I am not sure if it's a good idea because I am defining the interface of one component and it's only coincidence (my laziness) that the interface is almost the same. I will think about this tomorrow with fresh head... 🙂